### PR TITLE
tcpreplay: add page

### PR DESCRIPTION
**Pull request by jmckinny**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
```
tcpreplay version: 4.4.4 (build git:v4.4.4) (debug)
Copyright 2013-2022 by Fred Klassen <tcpreplay at appneta dot com> - AppNeta
Copyright 2000-2012 by Aaron Turner <aturner at synfin dot net>
The entire Tcpreplay Suite is licensed under the GPLv3
Cache file supported: 04
Compiled against libdnet: 1.17.0 (libdumbnet)
Compiled against libpcap: 1.10.4
64 bit packet counters: enabled
Verbose printing via tcpdump: enabled
Packet editing: disabled
Fragroute engine: enabled
Injection method: PF_PACKET send()
Not compiled with netmap
```

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thank you for writing the page :+1:

File: pages/common/tcpreplay.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# tcpreplay
+
+> Replay network traffic stored in pcap file.
```
Comment (spageektti) [inline pages/common/tcpreplay.md:3]:
```suggestion
> Replay network traffic stored in a `pcap` file.
```

File: pages/common/tcpreplay.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- List available network interfaces:
+
+`tcpreplay --listnics`
+
+- Replay traffic to interface:
+
+`tcpreplay -i {{eth0}} {{traffic.pcap}}`
+
+- Replay traffic to interface and stdout:
```
Comment (spageektti) [inline pages/common/tcpreplay.md:14]:
```suggestion
- Replay traffic to interface and `stdout`:
```

**Review by spageektti — COMMENTED** [review]
Hi. Thank you so much for your contribution. I wish you a wonderful christmas time!

**Review by spageektti — APPROVED** [review]

File: pages/common/tcpreplay.md (hunk 2 above)
Comment (jmckinny) [reply pages/common/tcpreplay.md]: Good catches, thanks!

**Pull request merged**